contract.go

```diff
@@ -44,6 +45,8 @@ var (
 
 	_ exec.ImportResolver = (*ContractExecutor)(nil)
 	_ compiler.GasPolicy  = (*ContractExecutor)(nil)
+
+	zeroPage = make([]byte, PageSize)
```

Comment:
Care to add the zero page optimization into another PR?

Reply:
Also, care to declare `zeroPage` into `common.go` instead?

Reply:
since this pr is closed i'll move all this whole optimization to separate pr